src/java.base/share/classes/java/time/ZoneId.java

```diff
@@ -186,15 +186,12 @@ public abstract sealed class ZoneId implements Serializable permits ZoneOffset,
      * This map allows the IDs to continue to be used via the
      * {@link #of(String, Map)} factory method.
      * <p>
-     * This map contains a mapping of the IDs that is in line with TZDB 2005r and
+     * This map contains a mapping of the IDs that is in line with TZDB 2024b and
      * later, where 'EST', 'MST' and 'HST' map to IDs which do not include daylight
-     * savings.
+     * savings since 1970.
```

Comment:
Hi Johny, the future flexibility clause seems to be missing in the spec update of ZoneId.

Reply:
Thanks for the changes in https://github.com/openjdk/jdk/pull/21265/commits/c3238444b63cb87c61f2503cbb1b5b7452181533.

Can we also add the following to the ZoneId specification,

```suggestion
     * savings since 1970. This mapping may change in update releases in support of new versions of TZDB.
```